tests/scripts/all.sh

```diff
@@ -1963,6 +1964,7 @@ component_test_when_no_ciphersuites_have_mac () {
     scripts/config.py unset MBEDTLS_CIPHER_NULL_CIPHER
     scripts/config.py unset MBEDTLS_ARC4_C
     scripts/config.py unset MBEDTLS_CIPHER_MODE_CBC
+    scripts/config.py unset MBEDTLS_ECJPAKE_C
```

Comment:
Perhaps instead of modifying ` tests/suites/test_suite_cmac.function`, this would suffice?
```
    scripts/config.py unset MBEDTLS_CMAC_C
```

Reply:
No, I don't think so: there's no reason why `MBEDTLS_CMAC_C` would be incompatible with this test. If there's a failure here, it means there's a problem that we hadn't detected until now.

Reply:
On second thoughts, if the purpose of the test includes having `MBEDTLS_CIPHER_MODE_CBC` and CMAC requires CBC, then it makes sense to turn off CMAC here. See https://github.com/ARMmbed/mbedtls/pull/4397/files#r622216536

Reply:
Done